src/Expression.php

```diff
@@ -77,6 +77,13 @@ class Expression implements \ArrayAccess, \IteratorAggregate
      */
     public $connection;
 
+    /**
+     * Wrap the expression in parentheses when consumed by another expression or not.
+     *
+     * @var bool
+     * */
+    public $wrapInParentheses = false;
```

Comment:
I think, if declared in Expression, we can make in protected and no need for BC at all

Reply:
Not really as we need access externally. See below:

https://github.com/atk4/dsql/blob/519410d9fb52fc238eab66dc1117a1f84230b4bf/tests/QueryTest.php#L684-L693

Reply:
you are right